interface/reports/receipts_by_method_report.php

```diff
@@ -23,12 +23,14 @@
 
 require_once("../globals.php");
 require_once("$srcdir/patient.inc");
-require_once "$srcdir/options.inc.php";
+require_once("$srcdir/options.inc.php");
 require_once("../../custom/code_types.inc.php");
 
 use OpenEMR\Common\Acl\AclMain;
 use OpenEMR\Common\Csrf\CsrfUtils;
 use OpenEMR\Core\Header;
+use OpenEMR\Services\InsuranceCompanyService;
+use OpenEMR\Services\InsuranceService;
```

Comment:
nice!
Services are taking over the world! (ok, a bit dramatic)

![2DV](https://user-images.githubusercontent.com/278968/88011412-1accf400-cacc-11ea-8a6b-899d573890f9.gif)